public_html/wp-content/mu-plugins/camptix-tweaks/addons/class-payment-options.php

```diff
@@ -93,6 +94,7 @@ function generate_payment_options( $payment_output, $total, $payment_methods, $s
 			<br class="tix-clear" />
 		</div>
 		<?php
+		return ob_get_clean();
```

Comment:
`MUST` - Nice catch! This should be a separate commit when merged to SVN.

Reply:
:+1: